src/motion/STK8XXXSensor.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+#ifndef _STK8XXX_SENSOR_H_
+#define _STK8XXX_SENSOR_H_
+
```

Comment:
Should there be an #endif here, rather than at the end of the file?

Reply:
Hi - yep good question - what I'm concerned about is making sure that each sensor only gets compiled and linked once

The #ifndef is an [include guard](https://includeguardian.io/article/pragma-once-vs-ifndef) to prevent the CPP header file being imported and compiled twice i.e. that's the reason the #if <-> #endif wraps around most of the whole .h file 

In theory #pragma once also does this, but to be honest I can't actually tell if the build chain for platform.io respects the #pragma once directive on all platforms, so am playing it safe and throwing them in.

Reply:
Thanks for the explanation :)